server/src/test/scala/controllers/application/ExportPortConfigControllerSpec.scala

```diff
@@ -89,8 +77,8 @@ class ExportPortConfigControllerSpec extends PlaySpec {
     val module = new DRTModule() {
       override val isTestEnvironment: Boolean = true
       lazy override val mockDrtParameters: MockDrtParameters = new MockDrtParameters() {
-        override val gateWalkTimesFilePath = Some(getClass.getClassLoader.getResource("gate-walktime-test.csv").getPath)
-        override val standWalkTimesFilePath = Some(getClass.getClassLoader.getResource("stand-walktime-test.csv").getPath)
+        override val gateWalkTimesFilePath = Some(getClass.getClassLoader.getResource("gate-walkTime-test.csv").getPath)
+        override val standWalkTimesFilePath = Some(getClass.getClassLoader.getResource("stand-walkTime-test.csv").getPath)
```

Comment:
These file names are a little funny with a mix of snake case and camel case. How about adding a hyphen between 'walk' and 'time', and lower-casing 'time'?